Add `New Featured Artist: Sparxe` news post (#10828)
ppy/osu-wiki: news/2024/2024-01-20-new-featured-artist-sparxe.md

---
layout: post
title: "New Featured Artist: Sparxe"
date: 2024-01-20 19:00:00 +0000
---

Homegrown musician **Sparxe** makes their way from osu!'s leaderboards to osu!'s Featured Artist listing!

![](https://assets.ppy.sh/artists/398/header.jpg)

> **Sparxe**'s music has been at the forefront of the competitive side of osu!taiko for years. From [the highest osu!taiko pp score](https://osu.ppy.sh/scores/taiko/173764399) to 11 [osu! originals](/wiki/osu!_originals) for various tournaments, practically everyone invested in osu!taiko knows **Sparxe**.

Reviewer: ```suggestion
**Sparxe**'s music has been at the forefront of the competitive side of osu!taiko for years. From [the highest osu!taiko pp score](https://osu.ppy.sh/scores/taiko/173764399) to 17 [osu! originals](/wiki/osu!_originals) (11 of which were for various tournaments), practically everyone invested in osu!taiko knows **Sparxe**.
```

> **Sparxe**'s music has been at the forefront of the competitive side of osu!taiko for years. From [the highest osu!taiko pp score](https://osu.ppy.sh/scores/taiko/173764399) to 17 [osu! originals](/wiki/osu!_originals) (11 of which were for various tournaments), practically everyone invested in osu!taiko knows **Sparxe**.

Reviewer: ```suggestion
**Sparxe**'s music has been at the forefront of the competitive side of osu!taiko for years. From [the highest osu!taiko pp score](https://osu.ppy.sh/scores/taiko/173764399) to 17 [osu! originals](/wiki/osu!_originals) (11 most of which were for various tournaments), practically everyone invested in osu!taiko knows **Sparxe**.
```

**6** of those tunes are at your fingertips (assuming you touch your screen after heading to [**Sparxe**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/398)).
 

> **Sparxe** specializes in an interpretive mix of ZUN's Touhou themes and Camellia's... Camellia-ness. Hear what that sounds like for yourself through this video of *Zinnober* and the rest of these previews:

Reviewer: ```suggestion
**Sparxe** specialises in an interpretive mix of ZUN's [i]Touhou[/i] themes and Camellia's... Camellia-ness. Hear what that sounds like for yourself through this video of *Zinnober* and the rest of these previews:
```
